wls-gui-wahllokalsystem/src/components/common/icons/TheBeanstandeteWahlbriefeRowStatusIcon.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-icon
+    :index="index"
+    :icon="getIcon()"
+    variant="text"
+    :color="getColor()"
+    :data-test="`rowstatus-icon-${index}`"
+  />
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from "pinia";
+import { VIcon } from "vuetify/components";
+
+import { useWahlenStore } from "@/stores/wahlenStore.ts";
+
+const { wahlen } = storeToRefs(useWahlenStore());
+
+const props = defineProps<{
+  index: number;
+}>();
+
+function getIcon() {
+  return _areRowInputsValid() ? "$valid" : "$edit";
+}
+
+function getColor() {
+  return _areRowInputsValid() ? "success" : "error";
+}
+
+function _areRowInputsValid() {
+  return wahlen.value
+    ? wahlen.value.every(
+        (wahl) =>
+          wahl.beanstandeteWahlbriefe &&
+          wahl.beanstandeteWahlbriefe[props.index] &&
+          !!wahl.beanstandeteWahlbriefe[props.index]
```

Comment:
Wofür ist die dritte Bedingung? Die Komponententests laufen auch ohne

```suggestion
          wahl.beanstandeteWahlbriefe[props.index]
```

Reply:
done [b504fb4](https://github.com/it-at-m/Wahllokalsystem/pull/1563/commits/b504fb40cf6913626f659a171cb0ac1ecd589c7b)